src/main/resources/db/changelog/changes/12-insert-users.yaml

```diff
@@ -18,6 +18,9 @@ databaseChangeLog:
               - column:
                   name: last_name
                   value: "Dylan"
+              - column:
+                  name: is_deleted
+                  valueNumeric: 0
```

Comment:
We don't need this. It should set 0 by default. Look how it is in mine:
![image](https://github.com/jv-feb24-team04/task-management-app/assets/83081198/2b0d223c-b892-4e51-a0fe-10117d540588)

Reply:
i don't know why it just has started without this column